sensu/files/conf.d/metrics_pure_storage.json

```diff
@@ -3,7 +3,7 @@
     "pure_storage_metrics": {
       "type": "metric",
       "command": "/opt/sensu/embedded/bin/metrics-pure-storage.py",
-      "handlers": [ "metrics" ],
+      "handlers": [ "metrics", "stackstorm" ],
       "interval": 60,
```

Comment:
I don't think the stackstorm handler should go in this file.   This one pulls data from PURE api and stores data in influxdb.